src/ClientInterface.php

```diff
@@ -298,6 +299,7 @@
  * @method int               xack(string $key, string $group, string ...$id)
  * @method string            xadd(string $key, array $dictionary, string $id = '*', array $options = null)
  * @method array             xautoclaim(string $key, string $group, string $consumer, int $minIdleTime, string $start, ?int $count = null, bool $justId = false)
+ * @method array             xclaim(string $key, string $group, string $consumer, int $minIdleTime, string|array $ids, ?XClaimOptions $options = null)
```

Comment:
I would say in this case additional abstraction around optional arguments is not the best approach, it should be as simple as possible so everyone could run it like this `$client->xclaim(...$arguments)`

Something like this would work better: 
`xclaim(... ?int idle = null, ?int $time = null, ?int count = null, bool $force = false, bool $justId = false, ?string $lastId = null)`

Reply:
Thanks for the review. I have made changes according to your recommendations.